src/transaction/transaction.rs

```diff
@@ -635,6 +635,11 @@ impl<PdC: PdClient> Transaction<PdC> {
         res
     }
 
+    /// Get the start timestamp of this transaction.
+    pub fn get_start_timestamp(&self) -> Timestamp {
```

Comment:
Does this more close to the Rust idiom?
```suggestion
    pub fn start_timestamp(&self) -> Timestamp {
```